pkg/helm/actions/kube_version.go

```diff
@@ -0,0 +1,30 @@
+package actions
+
+import (
+	"errors"
+	"k8s.io/client-go/discovery"
+	"k8s.io/client-go/rest"
+	"net/http"
+)
+
+func GetKubeVersion(apiEndpoint string, transport http.RoundTripper) (string, error) {
+	config := &rest.Config{
```

Comment:
why not passing to the function `rest.Config` instead?

Reply:
done